plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -84,6 +84,14 @@ func (t *TestAuthorizer) Authorize(a authorizer.Attributes) (authorized authoriz
 	return authorizer.DecisionNoOpinion, "", nil
 }
 
+// AlwaysAllowAuthorizer is an implementation of authorizer.Attributes
+// which always says yes to an authorization request.
+type AlwaysAllowAuthorizer struct{}
```

Comment:
I don't see a reason of make it public if it's being used only inside this file.

Reply:
It is in upper case because I want to make it consistent with `TestAuthorizer`

Reply:
Do I remember right that we even have such an Authorizer somewhere already?

Reply:
> Do I remember right that we even have such an Authorizer somewhere already?

Done. Use the one imported from `k8s.io/apiserver/pkg/authorization/authorizerfactory`